table/tables/cache.go

```diff
@@ -52,7 +43,6 @@ type cachedTable struct {
 	handle    StateRemote
 	totalSize int64
 
-	lockingForRead tokenLimit
```

Comment:
Before removing this line, the `LockForRead` and `RenewLease` operation may happen at the same time ... but they share the same `StateRemote` handle instance for executing SQL.
That would cause DATA RACE